tests/test_importers/test_rdfs_importer.py

```diff
@@ -1,6 +1,17 @@
 # -*- coding: utf-8 -*-
 
-"""Test the module can be imported."""
+# Monkey patching jsonobj to fix windows issue
```

Comment:
Maybe add a note that this is only required untill linkml-runtime gets fixed: https://github.com/linkml/linkml-runtime/pull/391

Instead of this monkey patch, I suggest to just skip the test on windows by marking it with

```
@pytest.mark.skipif(sys.platform == "win32", reason="Does not run on windows untill linkml-runtime is fixed (https://github.com/linkml/linkml-runtime/pull/391)")
```

Reply:
I worry about missing something else in the test during the window that we aren't testing due to waiting on linkml-runtime. I'm probably be incredibly defensive in worrying about that. Either way, we have to remove these as soon as we migrate to the new linkml-runtime.

Reply:
To be clear... you're probably right that this is the better simpler solution but since I've already done the monkey-patch I'm inclined to leave it in defensively.